src/player.cpp

```diff
@@ -2992,9 +2996,12 @@ void player::disp_status( const catacurses::window &w, const catacurses::window
 
         //~ Movement type: "walking". Max string length: one letter.
         const auto str_walk = pgettext( "movement-type", "W" );
+        //~ Movement type: "sneaking". Max string length: one letter.
+        const auto str_sneak = pgettext( "movement-type", "S" );
         //~ Movement type: "running". Max string length: one letter.
         const auto str_run = pgettext( "movement-type", "R" );
-        wprintz( w, c_white, " %s", move_mode == "walk" ? str_walk : str_run );
+        wprintz( w, c_white, " %s", move_mode == "walk" ? str_walk :
```

Comment:
Seems like `move_mode` should be an enumeration.

Reply:
Yeah, thought so myself, but changing it would break old savegames?